packages/dcos-metrics/build

```diff
@@ -35,7 +35,7 @@ mkdir -p /pkg/src/github.com/dcos
 mv /pkg/src/dcos-metrics /pkg/src/github.com/dcos/dcos-metrics
 
 pushd /pkg/src/github.com/dcos/dcos-metrics
-make build
+make rawbuild
```

Comment:
It looks odd. Usually `make build` should work for everybody

Reply:
@philipnrmn ? ^

Reply:
`make build` uses docker. Docker isn't available in pkgpanda's default build env. `rawbuild` builds without docker. This resolves [DCOS-22194](https://jira.mesosphere.com/browse/DCOS-22194), linked in the PR description.